<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>35</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Start Real Time Update</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="screenLayout">
      <item>
       <widget class="QLabel" name="screenLabel">
        <property name="text">
         <string>Screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="ledCountLayout">
      <item>
       <layout class="QHBoxLayout" name="HledCountLayout">
        <item>
         <widget class="QLabel" name="HledCountLabel">
          <property name="text">
           <string>Horizontal LEDs Count:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="HledCountSpinbox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="VledCountLayout">
        <item>
         <widget class="QLabel" name="VledCountLabel">
          <property name="text">
           <string>Vertical LEDs Count:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="VledCountSpinbox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
